src/native.props

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemDefinitionGroup>
-    <Link>
-      <ImageHasSafeExceptionHandlers>true</ImageHasSafeExceptionHandlers>
+    <Link Condition="'$(Platform)'=='x64'">
       <RandomizedBaseAddress>true</RandomizedBaseAddress>
```

Comment:
actually can remove the != 64 as that config is now gone. good note in any case, thanks

---
In reply to: [134349685](https://github.com/Microsoft/sarif-sdk/pull/590#discussion_r134349685) [](ancestors = 134349685)